ast/desugar/Desugar.cc

```diff
@@ -84,10 +84,11 @@ ExpressionPtr numparamTree(DesugarContext dctx, int num, parser::NodeVec *decls)
     return MK::Local(core::LocOffsets::none(), name);
 }
 
-ExpressionPtr node2TreeImpl(DesugarContext dctx, unique_ptr<parser::Node> what);
+ExpressionPtr node2TreeImpl(DesugarContext dctx, unique_ptr<parser::Node> what, bool calledFromExtractToVariable);
```

Comment:
We should put this in the `DesugarContext` which is already threaded to all of these calls.